### PR TITLE
Update FLIMfit to OMERO 5.2 & new docs link

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/eoY3P7cX/428-update-flimfit-downloads-page
Can be tested once @imunro is ready to release the new version.

### CODE REVIEW

**Comment by hflynn** [comment]
https://github.com/imperial-photonics/FLIMfit/issues/205 - suggestion to serve the MCR from our downloads artifacts too

**Comment by imunro** [comment]
Looks good to me

File: flimfitgen.py [hunk 1]
```diff
@@ -34,8 +35,8 @@ def usage():
     "/MCR_R2015b_maci64_installer.zip"
 
 for x, y in (
-        ("FLIMFIT_51_WIN", "artifacts/FLIMfit_@VERSION@_x64.zip"),
-        ("FLIMFIT_51_MAC", "artifacts/FLIMfit_@VERSION@_MACI64.zip"),
+        ("FLIMFIT_52_WIN", "artifacts/FLIMfit_@VERSION@_x64.zip"),
+        ("FLIMFIT_52_MAC", "artifacts/FLIMfit_@VERSION@_MACI64.zip"),
```
Comment (hflynn) [inline flimfitgen.py:39]: @sbesson if we wanted to host the MCR on downloads, would I just need to add the MCR locations here and delete the section above?

Comment (hflynn) [inline flimfitgen.py:39]: Okay, nevermind, I got the wrong end of the stick, it was changing the windows installer within the FLIMfit download to serve MCR from our server that was being suggested, not changing this link.

Comment (imunro) [inline flimfitgen.py:39]: I think we were discussing changing this link but then it was pointed out that the wIndows installer could also change.
The two changes aren't necessarily linked

Comment (hflynn) [inline flimfitgen.py:39]: I changed the filename to match these instead when I was trying to test this, but I can certainly amend them if you want to give me a new filename.

**Comment by imunro** [comment]
If adding something to the ci build e.g. checksum calculation would help simplify the proces please let me know.

File: flimfitgen.py (hunk 1 above)
Comment (imunro) [inline flimfitgen.py:39]: I have no strong feelings on the file names Whatever works for you but my guess would be that we're unlikely to return to releasing parallel versions for different OMERO's so the "_52" bit is probably superfluous.

**Pull request merged**